media/css/firefox/family/components/modules/_privacy.scss

```diff
@@ -19,4 +19,39 @@
             background-color: f3.$green-light;
         }
     }
+
+    @media #{$mq-sm} {
+        .l-grid {
+            display: grid;
+
+            .c-subtitle {
+                min-width: 8ch;
+                max-width: 12ch;
+            }
+        }
+    }
+
+    @media #{$mq-lg} {
+        .l-grid {
+            grid-template-columns: minmax(20ch, 35ch) 1fr;
+            grid-template-rows: 100px auto auto;
```

Comment:
👍 cool, good eye on including the module tag in the grid layout to get the right alignment in the second column